### DISCUSSION TITLE
ASAM OSI Layout

### DISCUSSION THREAD

**Discussion by jdsika** [lichtblick-suite/lichtblick#216] [Ideas]
The moment we have the two public repositores for the ASAM OSI panel and converter we should add a specific layout two work with these two extensions.

**Comment by carolinabamorim** [comment]
Hi @jdsika, as suggested I believe the ASAM OSI panel and converter should move out of our project. I believe this is not related to Lichtblick and it shouldn't be in our discussions.
Thank you.
